# HG changeset patch
# User Marco Atzeri <marco_atzeri@yahoo.it>
# Date 1270752397 14400
#      Thu Apr 08 14:46:37 2010 -0400
# Node ID cbf6eebb2a0bc20fe9b2cd21971ab1e2535631b8
# Parent  8f27f368aba21e6d9f6168a309649b1d6aba94fe
Makefile.am replace EXEEXT with SCRIPTS_EXE_SUFFIX for installing
and removing scripts

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2010-04-08  Marco Atzeri  <marco_atzeri@yahoo.it>
+
+	* Makefile.am (make-version-links, remove-version-links):
+	Use SCRIPTS_EXE_SUFFIX for scripts, not EXEEXT.
+
 2010-04-07  John W. Eaton  <jwe@octave.org>
 
 	* README.ftp, README.devel:
 	Update from Judd Storrs <jstorrs@gmail.com>.
 
 2010-03-27  David Bateman  <dbateman@free.fr>
 
 	* acinclude.m4 (OCTAVE_CHECK_OPENMP): Macro to check for OpenMP support
diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -186,19 +186,19 @@ install-data-local: installdirs-local
 
 install-exec-hook: make-version-links
 
 uninstall-local: remove-version-links
 
 make-version-links:
 	cd $(DESTDIR)$(bindir) && \
 	for f in $(basename $(bin_SCRIPTS)); do \
-	  mv $$f$(EXEEXT) $$f-$(version)$(EXEEXT) && \
-	    $(LN_S) $$f-$(version)$(EXEEXT) $$f$(EXEEXT); \
+	  mv $$f$(SCRIPTS_EXE_SUFFIX) $$f-$(version)$(SCRIPTS_EXE_SUFFIX) && \
+	    $(LN_S) $$f-$(version)$(SCRIPTS_EXE_SUFFIX) $$f$(SCRIPTS_EXE_SUFFIX); \
 	done
 .PHONY: make-version-links
 
 remove-version-links:
 	for f in $(basename $(bin_SCRIPTS)); do \
-	  rm -f $(DESTDIR)$(bindir)/$$f-$(version)$(EXEEXT); \
+	  rm -f $(DESTDIR)$(bindir)/$$f-$(version)$(SCRIPTS_EXE_SUFFIX); \
 	done
 .PHONY: remove-version-links
 
